### PR TITLE
[IMP] website_forum: bring forum modes

### PR DESCRIPTION
**Pull request by kea14**
It is now possible to specify the forum mode : Questions and Answers or
Discussions. In Q&A mode, a user can only reply once while there is no
limit in Discussions mode.

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/website_forum/static/src/js/website_forum.editor.js [hunk 1]
```diff
@@ -39,8 +40,9 @@ WebsiteNewMenu.include({
                         '    <input type="checkbox" required="required"/> '+
```
Comment (JKE-be) [inline addons/website_forum/static/src/js/website_forum.editor.js]: Doen't make sense to move this part to forum_template xml file too ?

**Comment by kea14** [comment]
updated as requested by @sbuhl

**Comment by kea14** [comment]
I can't lay hands on this error on runbot... :disappointed: Do you have an idea or could you glance at it when you have some time for me ? :see_no_evil:

File: addons/website_forum/static/src/xml/website_forum_templates.xml [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+        <div class="form-group mb0">
+            <label class="offset-md-3 col-md-9 text-left">
+                <input type="checkbox" required="required"/>
+            </label>
+        </div>
+    </t>
+    <t t-name="website_forum.select_forum_mode">
+        <div class="form-group row mb0">
+            <label class="col-md-3 col-form-label">Forum Mode:</label>
+            <div class="form-check form-check-inline">
```
Comment (qsm-odoo) [inline addons/website_forum/static/src/xml/website_forum_templates.xml]: https://getbootstrap.com/docs/4.3/components/forms/#checkboxes-and-radios-1 -> we use the custom form elements not those one

Comment (qsm-odoo) [reply addons/website_forum/static/src/xml/website_forum_templates.xml]: + you can't mix cols and non-cols... you can see that the template is broken

**Comment by JKE-be** [comment]
+ inspire your code from webslide_slide create_new...and the dialog

and move xml_dependencies into the dialog

**Comment by kea14** [comment]
@JKE-be should be good now :)

**Comment by JKE-be** [comment]
please rename 'Send anwer', doesn't mean anything

an you also remove this button on discussion mode, no interest... It is a discussion, not a debate ;)

**Comment by kea14** [comment]
branch is up-to-date

**Comment by JKE-be** [comment]
branch is red :/

**Comment by JKE-be** [comment]
please, test at leas tlocally your modification ...
It is just broken during in normal process ! not a test...

**Comment by kea14** [comment]
Testing is not the problem but my bad I should have rebased before my modification :lol, btw why 'active' is not accessible anymore in the view ? I'll check that this evening and re push

**Comment by JKE-be** [comment]
already done, an active invisible="1" that you remove

You cannot install forum after your rebase

**Comment by kea14** [comment]
Oh ok. Yes sorry for that :/ thanks for your help

**Comment by JKE-be** [comment]
@robodoo r+

**Pull request closed without merging**